src/forms/EditStudyForm.js

```diff
@@ -1,255 +1,155 @@
-import React from 'react';
-import {Form, Label, Message, Grid} from 'semantic-ui-react';
+import React, {useState, Fragment} from 'react';
+import {Form} from 'semantic-ui-react';
+import {PopupFormField} from '../components/PopupFormField';
 
 const EditStudyForm = ({formikProps}) => {
```

Comment:
not sure a separate form is necssary, I think we could reuse the NewStudyForm, let's talk. I think we maybe able to use a `forwardRef` to the form element and get around having to duplicate alot of the form code